01_todo_app/web/src/components/landing/index.jsx

```diff
@@ -1,68 +1,133 @@
+                                    />
+
+                                    <button
+                                        type="submit"
+                                        className="relative -ml-px inline-flex items-center space-x-2 rounded-r-md text-white text-lg bg-button px-8 md:px-24 py-2 "
+                                    >
+                                        Add
+                                    </button>
+                                </div>
+                            </div>
+
+                            <div>
+                                <input type="hidden" value="NEW" name="status" />
+                            </div>
+                        </form>
+                    </div>
+                    <div className="grid grid-cols-12 mt-20 gap-x-8 md:gap-x-0 gap-y-10">
+                        {messages.map((message) => (
+                            <>
+                                <div className="col-span-1">
+                                    <input
+                                        type="checkbox"
+                                        value=""
+                                        name="checked"
+                                        onChange={(event) => updateTodo(event, message)}
+                                        checked={message.status === "DONE"}
+                                        className="bg-white border border-gray-400 rounded w-6 h-6 text-black p-0 accent-checked"
+                                    />
+                                </div>
+                                <div className="col-span-7 break-words md:col-span-9 text-white">
+                                    <span className={`${message.status == "DONE" ? "line-through" : ""}		text-base`}>{message.text}</span>
+                                </div>
+                                <div className="col-span-1"></div>
+                                <div className="col-span-3 md:col-span-1 flex justify-end">
+                                    <svg
+                                        xmlns="http://www.w3.org/2000/svg"
+                                        fill="none"
+                                        viewBox="0 0 24 24"
+                                        stroke-width="1.5"
+                                        stroke="currentColor"
+                                        class="w-5 h-5 text-gray-500 border border-gray-500 cursor-pointer hover:text-red-500 hover:border-red-500"
+                                        onClick={(event) => deleteTodo(event, message)}
+                                    >
+                                        <path stroke-linecap="round" stroke-linejoin="round" d="M19.5 12h-15" />
+                                    </svg>
+
+                                    <form onSubmit={(event) => deleteTodo(event, message)}></form>
```

Comment:
Why do we have an empty form here? It has no child elements at all...

Reply:
Not my work

Reply:
But either way it just appears to be something that shouldn't be rendered for the user but what makes the delete functionality work.

Reply:
Nvm, looked inside my iDE and I can see the SVG has the same onClick event. This is probably something on my end. I take my words back :-)